### PR TITLE
feat: add existingSecret value to override secret with ext secrets

### PR DESCRIPTION
**Pull request by josephvano**
* add helper methods to create secret
* add helper method to get secre name
* update refs for secret to use helper method

Signed-off-by: Joseph Vano <joseph.vano@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by Jasstkn — CHANGES_REQUESTED** [review]
@josephvano hi! thank you for your contribution!

To start review process, please, update your branch from the master and resolve any conflicts on the way.

**Comment by Jasstkn** [comment]
> Hi @Jasstkn I've rebased from master. Thanks

I see you have a failed job about updating docs. Did you check our contributing guide?
https://github.com/litmuschaos/litmus-helm/blob/master/CONTRIBUTING.md#how-to-contribute

**Comment by josephvano** [comment]
Hi @Jasstkn I've updated the helm-docs and rebased again.

I'm not sure why there is an error with the `lint-charts` check with validation of maintainers.

**Comment by Jasstkn** [comment]
> Hi @Jasstkn I've updated the helm-docs and rebased again.
> 
> I'm not sure why there is an error with the `lint-charts` check with validation of maintainers.

Thanks. I'm fixing the issue right now. :)

**Review by Jasstkn — APPROVED** [review]
Hi @josephvano! Again thank you for your contribution. LGTM :)

**Comment by Jasstkn** [comment]
@ispeakc0de @imrajdas Hi folks. Please, review this PR. :)

**Comment by Jasstkn** [comment]
> @ispeakc0de @imrajdas Hi folks. Please, review this PR. :)

@ispeakc0de @imrajdas ping

**Comment by agcorrea** [comment]
+1

**Comment by josephvano** [comment]
Rebased

ready for review again @ispeakc0de @imrajdas

**Review by imrajdas — APPROVED** [review]

File: charts/litmus/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "2.10.0"
 description: A Helm chart to install ChaosCenter
 name: litmus
-version: 2.10.2
+version: 2.11.0
```
Comment (Jonsy13) [inline charts/litmus/Chart.yaml:5]: Can we please change it to **`2.10.3`**

File: charts/litmus/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # litmus
 
-![Version: 2.10.2](https://img.shields.io/badge/Version-2.10.2-informational?style=flat-square) ![AppVersion: 2.10.0](https://img.shields.io/badge/AppVersion-2.10.0-informational?style=flat-square)
+![Version: 2.11.0](https://img.shields.io/badge/Version-2.11.0-informational?style=flat-square) ![AppVersion: 2.10.0](https://img.shields.io/badge/AppVersion-2.10.0-informational?style=flat-square)
```
Comment (Jonsy13) [inline charts/litmus/README.md:3]: Can we please change it to **`2.10.3`**

**Review by Jonsy13 — CHANGES_REQUESTED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**